nix/modules/release-automation.nix

```diff
@@ -160,7 +160,7 @@
               --force-tag-creation \
               --force-branch-creation \
               --additional-manifests="crates/test_utils/wasm/wasm_workspace/Cargo.toml" \
-              --allowed-semver-increment-modes="!pre_patch beta-dev" \
+              --disallowed-version-reqs=">=0.4" \
```

Comment:
```suggestion
--allowed-semver-increment-modes="!pre_patch beta-dev"
```

Reply:
Follow up PR coming to switch to `beta-rc`. I will include this change